Lib/glyphsLib/builder/features.py

```diff
@@ -68,6 +68,8 @@ def to_ufo_features(self, ufo):
 
 def _build_gdef(ufo):
     """Build a table GDEF statement for ligature carets."""
+    import glyphsLib.glyphdata  # Expensive import
```

Comment:
I think `from glyphsLib import glyphdata` would be better. it spares you a extra attribute lookup

Reply:
ok